### PR TITLE
Content block segmented content model

### PR DESCRIPTION
**Pull request by mfearer94**
This follows the new content model layout as the 'content-group-simple' because the content item component page does not exist. I've added an anchor link to this page and added security to the outgoing Box link in the design and functional specifications section.

### Changelog

**New**

- Content model
- Anchor link

**Changed**

- Added security to existing box link

### CODE REVIEW

**Comment by JennySanchez** [comment]
Hi @mfearer94 Are the items in the "Sub-component field" under the "Content item"? 

Screen Shot 2021-02-10 at 2 37 20 PM

**Comment by mfearer94** [comment]
@JennySanchez They are! Hopefully that was clear cause if not, we will want to change this around a bit

File: src/pages/components/content-block-segmented.mdx [hunk 1]
```diff
@@ -32,9 +33,26 @@ This variation includes navigational quick links on the right side. These quick
 
 ![Image of content block simple with aside elements](../../images/pattern/content-block-segmented/content-block-segmented-with-aside-elements.jpg)
 
+## Content guidance
+
+| Field | Sub-component field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ---------- | ---------- |
+| Headline | | Short copy | Yes | 1 | XS (65 / 85)|
+| TOC headline override | | Short copy | No | 1 | Mini (25 / 35)|
```
Comment (photodow) [inline src/pages/components/content-block-segmented.mdx:41]: Lol, I am drawing a blank on this. Sorry.

File: src/pages/components/content-block-segmented.mdx [hunk 2]
```diff
@@ -32,9 +33,26 @@ This variation includes navigational quick links on the right side. These quick
...
+| Paragraph | | Long copy | Yes | 1 | XXL (350 / 450)|
+| [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) or [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) | | Component | No | 1 | - |
```
Comment (photodow) [inline src/pages/components/content-block-segmented.mdx:43]: Calling out content block for what it's worth.

**Review by photodow — COMMENTED** [review]
TOC please? :D

File: src/pages/components/content-block-segmented.mdx [hunk 3]
```diff
@@ -32,9 +33,26 @@ This variation includes navigational quick links on the right side. These quick
...
+| Content item | | Component | Yes | 2-3 | - |
+| | Headline | Short copy | Yes | 1 | XS (65 / 85) |
+| | Paragraph | Long copy | Yes | 1 | XXXXL (1600 / 2400) |
+| | [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) | Component | No | 1 | - |
+| [CTA (card)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | | Component | No | 1 | - |
+| [Link list](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/link-list) | | Component | No | 1 | - |
```
[the remark this answers is not in the document]
Comment (mfearer94) [reply src/pages/components/content-block-segmented.mdx]: @photodow Just updated. Let me know what you think.

[the remark this answers is not in the document]
Comment (mfearer94) [reply src/pages/components/content-block-segmented.mdx]: @photodow Need your thoughts on the updated content

**Comment by mfearer94** [comment]
@JennySanchez This is ready for your final review.

**Review by JennySanchez — APPROVED** [review]
Looks great!

**Pull request merged**